docker/demo/config/bench/complex-workflow-dag-mor.yaml

```diff
@@ -0,0 +1,104 @@
+first_insert:
```

Comment:
Feel like adding more recipes and comments - like for single insert/bulk-insert/update would encourage folks ramping up on hudi to compose their own dag and read the code-path to better understand

Reply:
Yes, that's the plan. If you think we can beef this particular file more, I can do that. For adding more recipies, I'm hoping you guys start using it for your tests and add more complex recipies.

Reply:
If people are ramping on Hudi by writing tests, then I am not sure how to feel about this. :) This is mostly for Hudi devs to develop in confidence.. 

@n3nash on this topic, it would be great if you can change (if they are not already proven to do this) these recipes to get us coverage.. I am interested to know for .eg how many bugs you found running these dags. and for how long? Even if you can setup some additonal CI runs to keep running these tests on different commits/ or on CircleCI/Azure pipelines, that would complete the usefulness of this.

Reply:
@vinothchandar Did not completely follow your comment about `ramping up by writing tests` ?

So, I've added some recipies to the unit tests of the TestSuiteJob which will run as part of the build. Some of these other recipies are examples (which I have validated) for folks to use. The intent is for these recipies to be used for initial understanding of hoodie and for testing of large features. 
The next step, as you mentioned, is to have a longer running CI job that runs this recipe (which happens now as part of a docker step that I've added to the documentation PR : https://github.com/apache/incubator-hudi/pull/626).
So I have 2 proposals : 
1) We can either run this recipe as part of the integration test setup that runs all the docker steps
2) We can have a CI job that runs this recipe